glyphs2fontir/src/source.rs

```diff
@@ -320,16 +325,16 @@ impl Work for GlyphIrWork {
             .get(&self.glyph_name)
             .ok_or_else(|| WorkError::NoGlyphForName(self.glyph_name.clone()))?;
 
-        let mut sources = HashMap::new();
+        let mut ir_glyph = ir::Glyph::new(self.glyph_name.clone());
```

Comment:
in ufo2fontir::source::GlyphIrWork::exec() method, we dont create an ir::Glyph inline, but we call ufo2fontir::toir::to_ir_glyph function -- should we do the same for glyphs2fontir?

https://github.com/googlefonts/fontmake-rs/blob/66aa397c76b7d76ba9e9ed8d123dcff2eaacab80/ufo2fontir/src/source.rs#L452-L453

also note how ufo2fontir we map the error returned from to_ir_glyph as a `WorkError::GlyphIrWorkError` variant.

I noticed that glyphs2fontir has no errors.rs of its own, either it's infallible (which I doubt) or it is using the fontir::ir:errors namespace as its own.
I think we should only add to the latter generic errors that would apply to an font IRs in general. E.g. "NoLayerForGlyph" makes sense as a glyphs2fontir::error but not much as, say, ufo2fontir..

Reply:
Wherever errors are reasonable for more than a specific source format or we can make them "generic" +1 to moving them to fontir::Error. I can't think of an immediate example when that wouldn't be true but in abstract I wouldn't want to see fontir::Error::UfoSpecificSituation.

Reply:
I don't think I want to completely redo errors here, will file an issue.

Reply:
#64